### DISCUSSION TITLE
Diskless or Thin clients?

### DISCUSSION THREAD

**Discussion by crazy-logic** [ltsp/ltsp#304] [Q&A]
So in the readme it seems like LTSP is now a diskless client project rather than a true thin client project - is that the case? I have installed a LTSP server and got it to the client can get to login stage but it appears not to be a true thin client - is that the case?

**Comment by alkisg** [comment]
Hi, this is a duplicate of #243, read that one and if you still have questions let's continue there.

**Reply by BenoitMaisonny** [reply]
Issue 32 has been deleted...

**Comment by alkisg** [comment]
Some links were broken due to the conversion to discussions, I updated it now.

Github devs are working to fix the redirection links.
